arch/risc-v/src/qemu-rv/Make.defs

```diff
@@ -34,8 +34,7 @@ CMN_CSRCS += riscv_releasepending.c riscv_reprioritizertr.c
 CMN_CSRCS += riscv_releasestack.c riscv_stackframe.c riscv_schedulesigaction.c
 CMN_CSRCS += riscv_sigdeliver.c riscv_unblocktask.c riscv_usestack.c
 CMN_CSRCS += riscv_idle.c riscv_tcbinfo.c riscv_cpuidlestack.c
-CMN_CSRCS += riscv_exception.c riscv_getnewintctx.c riscv_doirq.c
-CMN_CSRCS += riscv_cpuindex.c
+CMN_CSRCS += riscv_fault.c riscv_getnewintctx.c riscv_doirq.c
```

Comment:
Is `riscv_fault.c` missing or that was supposed to be `riscv_exception`?

Reply:
Yes rebase error. riscv_fault does not exist anymore, need to fix

Reply:
Done